src/packages/console/src/pages/siteVersionPreviewPage.tsx

```diff
@@ -214,26 +212,30 @@ export const SiteVersionPreviewPage = (props: ISiteVersionPreviewPageProps): Rea
   });
 
   return (
-    <div className={classes.root}>
+    <React.Fragment>
       <Helmet>
         <title>{`${site ? site.name : 'Site page'} | Preview ${siteVersion ? siteVersion.name : ''} | Everypage Console`}</title>
       </Helmet>
       <NavigationBar />
-      <main className={classes.content}>
+      {/* TODO: Still need to position the container,
+       find a way to display the containing view just below the  navbar*/}
+      <Spacing variant={PaddingSize.Wide4}/>
+      <ContainingView>
         {site === null || siteVersion === null || siteVersionEntry === null ? (
-          <div>Error loading site version. Please go back and try again...</div>
+          <Text>Error loading site version. Please go back and try again...</Text>
         ) : siteContent === undefined || siteTheme === undefined || assetFileMap === undefined ? (
-          <div>Loading...</div>
+          <Text>Loading...</Text>
         ) : (
           <React.Fragment>
-            <Box paddingX={2} paddingY={1} className={classes.metaBox}>
-              <Typography variant='subtitle1'>
+            <Stack direction={Direction.Horizontal} contentAlignment={Alignment.Start} childAlignment={Alignment.Center} shouldAddGutters={true} defaultGutter={PaddingSize.Wide} paddingLeft={PaddingSize.Wide} paddingVertical={PaddingSize.Default}>
```

Comment:
move stack to be directly under the containing view

Reply:
![image](https://user-images.githubusercontent.com/42097538/105444669-4952a400-5c94-11eb-8933-44436b9ddad3.png)
Still the same problem
I think we need to add a custom margin or padding

Reply:
oh i wasnt trying to solve a problem, just general advice. This problem here is cos you shouldn't be using a containing view at all actually (it limits the width)